app/HutchAgent/Services/OpenIdIdentityService.cs

```diff
@@ -104,7 +103,8 @@ public async Task<(string identity, string access)> RequestUserTokens(OpenIdOpti
   /// <param name="username">The User's Username</param>
   /// <param name="password">The User's Password</param>
   /// <returns>The requested tokens</returns>
```

Comment:
The param names don't match the signature and we're not using the username or password with this method.

Reply:
nvm, I can't read lol